### ISSUE TITLE
webrec temp body file and disk space utilization

### ISSUE THREAD

**Issue by sparvu** [kronometrix/recording#99]
It has been observed systems running on webrec will end up having all disk space used over time. It seems there are two run away files which are created by webrec and their filenames removed from file system but the file descriptor reference opened. 

```
krmx@hki1:/opt/kronometrix/log/current$ lsof +L1 
COMMAND  PID USER   FD   TYPE DEVICE  SIZE/OFF NLINK   NODE NAME
webrec  7041 krmx    3u   REG  252,0    638976     0 259169 /tmp/xRDB_0hs7o (deleted)
webrec  7041 krmx    4u   REG  252,0 119799808     0 259307 /tmp/ylLKj6MORp (deleted)
```

**Comment by sparvu** [comment]
will be ASAP taken into consideration. This affects env where webrec is running eating all disk space.

**Comment by sparvu** [comment]
fixed in 1.1 branch
